packages/react/src/box/styles.ts

```diff
@@ -115,6 +115,12 @@ type TypographyProps = Partial<{
 	fontSize: ResponsiveProp<FontSize>;
 	lineHeight: LineHeight;
 	textAlign: ResponsiveProp<'left' | 'center' | 'right'>;
+	/**
+	 * Allow long words to wrap if there is not enough space to fit them.
+	 *
+	 * Note that all child elements will inherit this behavior.
```

Comment:
I'm curious why we need to call out inheritance? We don't do that anywhere else.

Reply:
Yeah, all the other typography things are inherited. It's CSS so people should know

Reply:
Removed the JS Doc comment